Assignments/Object oriented/Final_Project_food.js

```diff
@@ -0,0 +1,68 @@
+CB.CloudApp.init('yegguroxoyua', '47345f24-5127-4318-abd3-b054fe11a2a2');
+
+class food {
+	constructor(ingredients,day,name,price,sides,codeName){
+		this.ingredients = ingredients;
+		this.day = day;
+		this.name = name;
+		this.price = price;
+		this.sides = sides;
+		this.codeName = codeName;
+	}
+}
+
+var d = new Date();
+var weekday = new Array(7);
+weekday[0]=  "Sunday";
+weekday[1] = "Monday";
+weekday[2] = "Tuesday";
+weekday[3] = "Wednesday";
+weekday[4] = "Thursday";
+weekday[5] = "Friday";
+weekday[6] = "Saturday";
+
+var n = weekday[d.getDay()];
+console.log(n);
+
+//Saturday meals
+const steak = new food([Steak,Pepper,Salt],"Saturday","Steak",17,[Broccoli,Rice_Pilaf],"steak");
```

Comment:
One way to eliminate so much copying / pasting would be to create an object or JSON structure containing this data.

```json
{
  "Saturday": [{
     "foodItems": [Steak, Pepper, Salt],
     "cost": 17
   },  {...}],
  "Sunday": [
   ]
}
```

Then you can loop through it and create everything dynamically. This also makes it easier to adjust the menu on the fly.